microsoft/react-native-test-app
# fix(ios): replace QRCodeReader with custom implementation

### Description

Replaces [QRCodeReader](https://github.com/YannickL/QRCodeReader.swift) with our own implementation.

### Platforms affected


### Test plan

1. Build and run iOS test app
2. Shake the device and tap "Scan QR Code"
3. Verify that rotation works
4. Verify that QR code scanning still works:
    ![image](https://user-images.githubusercontent.com/4123478/147913574-f0f6b5ad-ebf3-4354-b283-e68bef20b3e1.png)
    `http://localhost:8081/index.bundle`
